src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/EmptyDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.Testing
+{
+    /// <summary>
+    /// Defines a <see cref="DiagnosticAnalyzer"/> which does not report any diagnostics.
+    /// </summary>
+#pragma warning disable RS1001 // Missing diagnostic analyzer attribute.
```

Comment:
I would add this to a suppression file as we do not intend to fix (as it is not a bug) this so the pragma is just noise

Reply:
I'm not a fan of suppression files. Any objection to using 'SuppressMessage' here so I can provide the explanation inline?